templates/tease-search.twig

```diff
@@ -42,7 +42,7 @@
 
 					<div class="search-result-tags top-page-tags">
 						{% for page_type in post.p4_page_types %}
-							<button class="search-result-item-head no-btn tag-item tag-item--main page-type" data-href="{{ fn('get_term_link', page_type.term_id) }}">{{ page_type.name|e('wp_kses_post')|raw }}</button>
+							<a class="search-result-item-head no-btn tag-item tag-item--main page-type" data-href="{{ fn('get_term_link', page_type.term_id) }}">{{ page_type.name|e('wp_kses_post')|raw }}</a>
```

Comment:
If we change the `data-href` to just `href`, we don't need the following script to open the link. https://github.com/greenpeace/planet4-master-theme/blob/2a5faa966c2ecbfba8eb983d1344dae4eba69330/assets/src/js/search.js#L58-L58

Reply:
Ugh, right, this misses the whole point! Amending...

Reply:
@Inwerpsel Updated!